src/index.ts

```diff
@@ -188,32 +188,41 @@ app.get(
 // Check that the user is allowed to access the endpoint
 // and then run the query
 // The endpoint must contain a version number and a query name
-// e.g. /analytics/v1/someQuery
+// e.g. /analytics/v1/1234/someQuery
+// where 1234 is the podcast id and someQuery is the name of the query
 app.get(
-    '/analytics/:version/:query',
+    '/analytics/:version/:podcastId/:query',
     async (req: Request, res: Response, next: NextFunction) => {
         try {
+            const accountId = res.locals.user.accountId
```

Comment:
just to be on the safe side, we should check if account id or podcast id is not set and throw an error